src/QMCWaveFunctions/Fermion/SlaterDet.h

```diff
@@ -185,9 +185,12 @@ class SlaterDet : public WaveFunctionComponent
   {
     constexpr LogValueType czero(0);
 
+    // This log_value_ is in the slater determinant, it's still around but not consistent anymore with the
```

Comment:
This code makes it consistent so it must not have been before. Where as if 
```c++
LogValue SlaterDeterminant::getLogValue()
{
  LogValue log_value(0);
  for(auto& det : dets_)
  {
    log_value += det.get_log_value()
  }
  return log_value;
}
```
this code is unecessary and there is never inconsistent state in the slater determinant.

Reply:
For the record I don't think we have a bug due to this.  But it would be possible, it would also be possible for someone to "cleverly" use the stale state in slater determinant to do a comparison to the current state which is exactly the kind of thing I don't want to have to maintain.

Reply:
I vaguely remember, at certain point, log_value needs to be recomputed and overwrite the old value to wipe out accumulated numerical error from accepts. It is worth adding an assert since there is an out-of-sync concern.

Reply:
I'm not really sure what you mean about the assert. Perhaps we can move discussion of this on to the next PR or review of the Batched delayed update in general. For now we just do as we have always done.